io.xolatgames.marknpaul.yml

```diff
@@ -0,0 +1,24 @@
+app-id: io.xolatgames.marknpaul
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: marknpaul
+finish-args:
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=x11
+  - --device=dri
+modules:
+  - name: Mark and Paul
```

Comment:
please to use name with spaces. it's bound to cause trouble at one point or the other.